product_docs/docs/pgd/5/repsets.mdx

```diff
@@ -13,7 +13,7 @@ nodes.
 Every PGD group creates a replication set with the same name as
```

Comment:
In the above, is a noun missing after "multi-master?

Reply:
topologies